src/ethereum/mod.rs

```diff
@@ -254,20 +275,28 @@ impl<TX> IntoBlock for Web3Block<TX> {
                 }
             },
             parent_hash: self.parent_hash,
+            uncles_hash: self.parent_hash,
```

Comment:
Why is this `self.parent_hash`?
If `uncle_hash` is not available, can it be `0x`?

Reply:
It's was wrongly mapped. Uncle hash is available. I have corrected it.